alerts/cluster-ingress-operator/HAProxyDown.md

```diff
@@ -0,0 +1,29 @@
+# HAProxyDown
+
+## Meaning
+
+This alert fires when metrics report that HAProxy is down.
+
+## Impact
+
+Access to routes will fail. It may cause a severe outage for critical applications.
+
+## Diagnosis
+
+- Check the router logs:
+```sh
+oc logs <router pod> -n openshift-ingress
+```
+
+- Check the events:
+```sh
+oc get events -n openshift-ingress
+```
+
+- Check the load on the system where the routers are hosted.
+
```

Comment:
There are so many other things they could check.  At least describe how to turn on access logs and prometheus metrics to look at:

#Container Threads: (We should see a fairly consistent value with some fluctuations based on load if healthy)
avg(container_threads{namespace='openshift-ingress', container='router'}) by (instance)

#Container Processes: (We should see a fairly consistent value with some fluctuations based on load if healthy)
avg(container_processes{namespace='openshift-ingress', container='router'}) by (instance)

I got this from https://access.redhat.com/solutions/5721381, but I'm not sure we sure link to an access article from a runbook.